mmv1/products/gkeonprem/product.yaml

```diff
@@ -0,0 +1,45 @@
+# Copyright 2022 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+!ruby/object:Api::Product
+name: Gkeonprem
+display_name: GKE On-Prem
```

Comment:
nit: https://cloud.google.com/anthos/clusters/docs/on-prem/1.3 "GKE on-prem" appears in official documentation

Reply:
Thanks for checking. I just checked and confirm "Anthos On-Prem" will be the official product name.